bundles/GoogleMarketingBundle/.github/ci/files/config/analytics.yaml

```diff
@@ -1,27 +1,27 @@
-services:
-    _defaults:
-        autowire: true
-        autoconfigure: true
-        public: false
-    #
-    # GOOGLE TAG MANAGER
-    #
-    Pimcore\Bundle\GoogleMarketingBundle\EventSubscriber\TrackingCodeSubscriber: ~
-
-    Pimcore\Bundle\GoogleMarketingBundle\Config\ConfigProvider: ~
-    Pimcore\Bundle\GoogleMarketingBundle\Config\Config:
-        factory: [ '@Pimcore\Bundle\GoogleMarketingBundle\Config\ConfigProvider', 'getConfig' ]
-
-    Pimcore\Bundle\GoogleMarketingBundle\Config\SiteConfigProvider: ~
-
-    Pimcore\Bundle\GoogleMarketingBundle\Tracker\Tracker:
-        calls:
-            - [ setLogger, [ '@logger' ] ]
-
-    # fallback locator which is used from the deprecated Pimcore\Google\Analytics class
-    pimcore.analytics.google.fallback_service_locator:
-        class: Symfony\Component\DependencyInjection\ServiceLocator
-        public: true
-        tags:
-            - { name: container.service_locator }
-
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+    #
+    # GOOGLE TAG MANAGER
+    #
+    Pimcore\Bundle\EcommerceFrameworkBundle\EventListener\Frontend\TrackingCodeSubscriber: ~
```

Comment:
so now `GoogleMarketingBundle` requires `EcommerceFrameworkBundle` to work? we can't install it independently..right?

Reply:
No, it's the other way round, Ecommerce requires GoogleMarketing 😊 But this was already the case.